### ISSUE TITLE
Getting Error Message No such file or directory: /snapshot/resin-cli/node_modules/denymount/bin/denymount

### ISSUE THREAD

**Issue by 13bmartens** [balena-io/balena-cli#977]
- **resin-cli version:** 7.10.6
- **Operating system and architecture:** MAC-OSX 10.14.1 Beta

Following the tutorial at https://resinos.io/docs/iot2000/gettingstarted/
sudo resin local configure ~/Downloads/resin.img shows error:

No such file or directory: /snapshot/resin-cli/node_modules/denymount/bin/denymount

Help would be greatly appreciated :)

**Comment by gpulido** [comment]
Hello,
I'm having the same issue with resin-cli version 10.0.1 on a Mac-osx Darwin Kernel Version 18.5.0: Mon Mar 11 20:40:32 PDT 2019; root:xnu-4903.251.3~3/RELEASE_X86_64 x86_64
Using Debug=1 I have the following output:
````
Error: spawn /snapshot/balena-cli/node_modules/denymount/bin/denymount ENOENT
    at exports._errnoException (util.js:1020:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:197:32)
    at onErrorNT (internal/child_process.js:376:16)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickDomainCallback (internal/process/next_tick.js:128:9)
```
Thanks in advance

**Comment by pdcastro** [comment]
@gpulido, what command line were you running? Was it `sudo balena local configure`? I am using the exact same version of MacOS as yours, and also using CLI version 10.0.1, and the following works for me without errors:
```
$ pwd
/Users/paulo

$ sudo balena local configure ~/Downloads/test-raspberrypi3-2.31.5+rev1-dev-v9.11.3.img
Password:
? Network SSID My_Wifi_Ssid
? Network Key super_secret_wifi_password
? Do you want to set advanced settings? No
Done!
```
Did you use the [standalone install](https://www.balena.io/docs/reference/cli/#standalone-install) zip file, or installed via npm?
I wonder what I would have to do in order to reproduce the error. Could it be the image file?!

**Comment by gpulido** [comment]
Hello,
I'm using the standalone balena-cli and using the macos terminal.
And the command is the one you mentioned using a fresh raspberrypi3 balena.img downloaded from the balena repositories. I added balena to the path using the suggestions from the readme:
```
sudo balena local configure ~/Downloads/balena.img
Password:
No such file or directory: /snapshot/balena-cli/node_modules/denymount/bin/denymount

Additional information may be available in debug mode. Prefix the command
line with DEBUG=1, i.e.:  DEBUG=1 balena ...
If you need help, don't hesitate in contacting our support forums at
https://forums.balena.io

For bug reports or feature requests, have a look at the GitHub issues or
create a new one at: https://github.com/balena-io/balena-cli/issues/

````

**Comment by pdcastro** [comment]
@13bmartens, @gpulido, this issue should be fixed in CLI v10.10.3 or later ([installation guide](https://github.com/balena-io/balena-cli/releases)). I was able to reproduce it with the standalone installer, and after the fix my tests now pass. If you have any problems, feel free to reopen it. Thank you for reporting it!

**Comment by gpulido** [comment]
@pdcastro  I have just tested and it worked.
Thank you!
